pkg/image/origins.go

```diff
@@ -253,9 +253,9 @@ func GenerateImageOrigins(linuxImagesFromArgs, targetImages, targetWindowsImages
 		fileContents = fileContents + k + " " + v + "\n"
 	}
 
-	originsFile, err := os.Create("rancher-image-origins.txt")
+	originsFile, err := os.Create("rancher-images-origins.txt")
 	if err != nil {
-		return fmt.Errorf("could not create rancher-image-origin.txt file: %w", err)
+		return fmt.Errorf("could not create rancher-images-origins.txt file: %w", err)
```

Comment:
Should we extract this to a const at this point?

Reply:
Yeah, I'll do it now